react/src/components/footer/SprkFooter.js

```diff
@@ -331,6 +333,8 @@ SprkFooter.propTypes = {
       ),
     }),
   ),
+  /** A node for App Store links. . */
+  appStoreLinks: PropTypes.node,
```

Comment:
I think this comment might of been a typo. I think its "An optional spot to pass in content for App Store Links." or something like that.